testsuite/tests/parsetree/source_jane_street.ml

```diff
@@ -855,7 +855,11 @@ Line 1, characters 23-36:
                            ^^^^^^^^^^^^^
 Error: This value is "local"
        because it is "stack_"-allocated.
-       However, the highlighted expression is expected to be "global".
+       However, the highlighted expression is expected to be "global"
+       because it is an allocation
+       which is expected to be in the parent region or "global"
+       because it is a function return value.
```

Comment:
Hmm... it's not a function return value. It is *contained in* a function return value. Maybe

```suggestion
       because it is used in function return value.
```

?

Reply:
This will be addressed in https://github.com/oxcaml/oxcaml/pull/4919 by explictly stating "an element in a tuple".

We can merge that PR first?